experimental/PropertyDDS/packages/property-shared-tree-interop/src/schemaConverter.ts

```diff
@@ -0,0 +1,329 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { assert } from "@fluidframework/common-utils";
+import {
+	fail,
+	FieldKinds,
+	FieldSchema,
+	ValueSchema,
+	EmptyKey,
+	SchemaBuilder,
+	FieldKindTypes,
+	Any,
+	TreeSchema,
+	LazyTreeSchema,
+	brand,
+	Brand,
+} from "@fluid-experimental/tree2";
+import { PropertyFactory } from "@fluid-experimental/property-properties";
+import { TypeIdHelper } from "@fluid-experimental/property-changeset";
+
+const nodePropertyType = "NodeProperty";
+const referenceGenericTypePrefix = "Reference<";
+const referenceType = "Reference";
+const basePropertyType = "BaseProperty";
+const booleanType = "Bool";
+const stringType = "String";
+const enumType = "Enum";
+const numberTypes = new Set<string>([
+	"Int8",
+	"Int16",
+	"Int32",
+	"Int64",
+	"Uint8",
+	"Uint16",
+	"Uint32",
+	"Uint64",
+	"Float32",
+	"Float64",
+	enumType,
+]);
+const primitiveTypes = new Set([...numberTypes, booleanType, stringType, referenceType]);
+
+type PropertyContext = Brand<"single" | "array" | "map" | "set", "PropertyDDSContext">;
```

Comment:
What is `PropertyDDSContext`? I don't see it included e.g. in the `isPropertyContext` type guard below.